### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by Cirat19 — COMMENTED** [review]
GJ!

**Review by KarolinaVlasova — CHANGES_REQUESTED** [review]
pay attention to the comments

File: cypress/e2e/registration.cy.js [hunk 1]
```diff
@@ -1,11 +1,70 @@
...
+
+    cy.get('#city').
+      type('{downArrow}{enter}');
+
+    cy.get('#submit').
+      click();
+
+    cy.get('#example-modal-sizes-title-lg').should('have.text', 'Thanks for submitting the form');
 
+    cy.contains('tr', user.firstName).should('be.visible');
```
[the remark this answers is not in the document]
Comment (kazhannnova) [reply cypress/e2e/registration.cy.js]: fixed

File: cypress/e2e/registration.cy.js [hunk 2]
```diff
@@ -1,11 +1,70 @@
...
+    cy.visit('/');
+    cy.url().should('include', 'automation-practice-form');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    })
   });
 
-  it('', () => {
+  it('should successfully register user', () => {
+    cy.get('[placeholder="First Name"]').
```
[the remark this answers is not in the document]
Comment (kazhannnova) [reply cypress/e2e/registration.cy.js]: fixed

**Review by NataliiaSolotva — APPROVED** [review]
GJ!

**Pull request closed without merging**